apps/astarte_realm_management_api/test/astarte_realm_management_api/triggers/triggers_test.exs

```diff
@@ -30,7 +30,7 @@ defmodule Astarte.RealmManagement.API.TriggersTest do
       {:ok, trigger} =
         attrs
         |> Enum.into(@valid_attrs)
-        |> RealmManagement.API.Triggers.create_trigger()
+        |> RealmManagement.API.Triggers.create_trigger("test")
```

Comment:
```elixir
def create_trigger(realm_name, attrs \\ %{})
```

I think the arguments are backwards, you should break the pipeline and call `RealmManagement.API.Triggers.create_trigger("test", params)` (in reality the tests below are not even executed, but at least we don't introduce new wrong code)